internal/commands/user/list/command_test.go

```diff
@@ -0,0 +1,192 @@
+package list
+
+import (
+	//"errors"
+
+	//"strings"
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestUserListSetup(t *testing.T) {
+	t.Run("Should construct a Realm client with the configured base url", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		profile.SetRealmBaseURL("http://localhost:8080")
+
+		cmd := &command{inputs: inputs{}}
+		assert.Nil(t, cmd.realmClient)
+
+		assert.Nil(t, cmd.Setup(profile, nil))
+		assert.NotNil(t, cmd.realmClient)
+	})
+}
+
+func TestUserListHandler(t *testing.T) {
+	testApp := realm.App{
+		ClientAppID: "eggcorn-abcde",
+		Name:        "eggcorn",
+	}
+
+	newMockClient := func() mock.RealmClient {
+		realmClient := mock.RealmClient{}
+		realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+			testApp.GroupID = filter.GroupID
+			testApp.ID = filter.App
+			return []realm.App{testApp}, nil
+		}
+		return realmClient
+	}
+
+	t.Run("Should find app users", func(t *testing.T) {
+		testUsers := []realm.User{{ID: "user1"}, {ID: "user2"}}
+
+		realmClient := newMockClient()
```

Comment:
it doesn't appear that `newMockClient` is used anywhere else.  if thats the case, i think we can just simply create the entire mock client directly, rather than leverage a helper function here.